api-docs/cloud/ref.yml

```
@@ -11161,26 +11516,85 @@ paths:
       summary: Add a member to an organization
       tags:
         - Organizations
+      x-codeSamples:
+        - label: cURL
+          lang: Shell
+          source: >
+            curl --request POST
+            "http://localhost:8086/api/v2/orgs/INFLUX_ORG_ID/members \
+              --header "Authorization: Token INFLUX_API_TOKEN" \
+              --header "Accept: application/json" \
+              --header "Content-Type: application/json" \
+              --data '{
+                  "id": "09cfb87051cbe000"
+                }'
   /api/v2/orgs/{orgID}/members/{userID}:
     delete:
+      description: >
+        Removes a member from an organization.
+
+
+        Use this endpoint to remove a user's member privileges from a bucket.
+        This
+
+        removes the user's `read` and `write` permissions from the organization.
```

```suggestion
        Use this endpoint to remove a user's `read` and `write` permissions for the organization.
```

If mine is correct, you can fix it here now and then fix it later in openapi.